<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResultsListWidgetClass</class>
 <widget class="QWidget" name="ResultsListWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>285</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ResultsListWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(50,50,50);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="1,8">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="MyLogoButton" name="btn_ControlVisiable" native="true"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_ReSet">
       <property name="minimumSize">
        <size>
         <width>62</width>
         <height>28</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>62</width>
         <height>28</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	border: 1px solid rgba(255, 255, 255, 0.2);
	border-radius: 5px;
	color:#999999;
}
QPushButton:hover
{
	background: #4A4A4A;
	border: 1px solid rgba(255, 255, 255, 0.2);
	border-radius: 5px;
}
QPushButton:pressed
{
	background: #787878;
	border: 1px solid rgba(255, 255, 255, 0.2);
	border-radius: 5px;
}</string>
       </property>
       <property name="text">
        <string>Reset</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>Resource/BtnReSet.png</normaloff>Resource/BtnReSet.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="widget_ControlVisual" native="true"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyLogoButton</class>
   <extends>QWidget</extends>
   <header>UI/MyWidget/MyLogoButton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
